### PR TITLE
Scheduled delivery of jobs cleanup

### PR DESCRIPTION
**Pull request by minglis**
Depends on 

- [x] https://github.com/alphagov/notifications-api/pull/635

Tidies up after the DB migrations in the scheduled job.

- migrates existing statues
- drops status column
- not null on job_status

### CODE REVIEW

**Comment by quis** [comment]
Rebased this against master now that #635 is merged. Previous head was 7b95981.

File: migrations/versions/0049_set_job_status.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+# revision identifiers, used by Alembic.
+revision = '0049_set_job_status'
+down_revision = '0048_job_scheduled_time'
+
+from alembic import op
+
+
+def upgrade():
+    op.execute("update jobs set job_status = status where job_status is null")
```
Comment (quis) [inline migrations/versions/0049_set_job_status.py]: What does `= status` mean here?

Comment (minglis) [inline migrations/versions/0049_set_job_status.py]: Sets one to the other. So notifications.job_status becomes the value of notifications.status.

**Comment by quis** [comment]
Tested upgrade and downgrade locally, works as expected. 👍

**Comment by servingUpAces** [comment]
👍  Fixed the merge conflicts and migration script conflicts. This is good to go now.

**Pull request merged**